README.md

```diff
@@ -1,70 +1,62 @@
-# Getting Started with Create React App
+# Bookstore
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+A project to keep track of books a user has read, using React and Redux.
 
-## Available Scripts
+## Screenshot of the page
 
-In the project directory, you can run:
+![Screenshot Profile Page](XXXXXXX)
 
-### `yarn start`
+## Project Decription
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+The Bookstore is a website, that displays a list of books with option: to add, remove and edit a book and track progress of reading. Website with two subpages: "Books" and "Categories.
 
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
+## Build With
 
-### `yarn test`
+- `React`
+- `Redux`
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+## Live Version Link
 
-### `yarn build`
+[Click Here!](XXXXXXXXXXXXXXXXxx)
```

Comment:
**Optional Changes**
- [ ] The README acts as a living documentation for any reviewer or contributor to the project. Let us try to always keep it up to date. If you don't have any live demo, I advice that we don't include a broken link for any reason. This makes our README file look unprofessional. The same applies for broken screenshots, thank you.